Makefile.Common

```diff
@@ -197,3 +198,8 @@ misspell-correction: $(TOOLS_BIN_DIR)/misspell
 .PHONY: moddownload
 moddownload:
 	$(GOCMD) mod download
+
+.PHONY: gci
+gci: $(TOOLS_BIN_DIR)/gci
+	@echo "running $(GCI)"
+	@$(GCI) write -s standard -s default -s "prefix(github.com/open-telemetry/opentelemetry-collector-contrib)" $(ALL_SRC_AND_DOC)
```

Comment:
Just a quick question on the use of `$(ALL_SRC_AND_DOC)` here. Since GCI is for go imports, what's the need for it to have it run against *.md & *.yaml files. Does  the utility just skip any file not a .go file?  Doesn't throw any warnings or errors with exit codes other than 0 if running against a non-go file?

Reply:
Yes, it didn’t fail for me at least.